core/src/iosMain/kotlin/com/revenuecat/purchases/kmp/Purchases.ios.kt

```diff
@@ -1,288 +1,370 @@
-@file:Suppress("EXTENSION_SHADOWED_BY_MEMBER")
+@file:Suppress("ACTUAL_ANNOTATIONS_NOT_MATCH_EXPECT")
 
 package com.revenuecat.purchases.kmp
 
 import cocoapods.PurchasesHybridCommon.RCCommonFunctionality
-import cocoapods.PurchasesHybridCommon.RCPurchases
 import cocoapods.PurchasesHybridCommon.RCStoreProduct
+import cocoapods.PurchasesHybridCommon.configureWithAPIKey
+import cocoapods.PurchasesHybridCommon.setAirshipChannelID
+import cocoapods.PurchasesHybridCommon.setOnesignalUserID
 import cocoapods.PurchasesHybridCommon.showStoreMessagesForTypes
+import com.revenuecat.purchases.kmp.models.BillingFeature
 import com.revenuecat.purchases.kmp.models.GoogleReplacementMode
 import com.revenuecat.purchases.kmp.models.PromotionalOffer
 import com.revenuecat.purchases.kmp.models.StoreMessageType
 import com.revenuecat.purchases.kmp.models.StoreProduct
 import com.revenuecat.purchases.kmp.models.StoreProductDiscount
 import com.revenuecat.purchases.kmp.models.StoreTransaction
 import com.revenuecat.purchases.kmp.models.SubscriptionOption
-
-public actual typealias Purchases = RCPurchases
-
-public actual var Purchases.finishTransactions: Boolean
-    get() = finishTransactions()
-    set(value) {
-        setFinishTransactions(value)
+import com.revenuecat.purchases.kmp.strings.ConfigureStrings
+import platform.Foundation.NSURL
+import cocoapods.PurchasesHybridCommon.RCDangerousSettings as IosDangerousSettings
+import cocoapods.PurchasesHybridCommon.RCPurchases as IosPurchases
+
+public actual class Purchases(private val iosPurchases: IosPurchases) {
```

Comment:
Do we need to make the constructor `private` as well? So it can't be instantiated like that... I see we do it in the android version, so not sure.

Reply:
Yes! Good catch! https://github.com/RevenueCat/purchases-kmp/pull/103/commits/be80594f5cbe89947109a46c111ea5a3f03344d7